src/utils/gameTimer.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { GameStartType, TimerType } from "..";
+import { frameToGameTimer } from "./gameTimer";
+
+describe("when calculating the in-game timer", () => {
+  it("should calculate the timer correctly when increasing", () => {
+    const increasing_settings = {
+      timerType: TimerType.INCREASING,
+      startingTimerSeconds: 0,
+    } as GameStartType;
```

Comment:
```suggestion
    };
```
If we pick the required fields as in my other suggestion we don't need these casts either.

Reply:
I haven't used Pick before - that's a nice feature and I agree that this works better